qiskit/basics/getting_started_with_qiskit_terra.ipynb

```diff
@@ -11,14 +11,25 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Getting Started with Qiskit Terra"
+    "# Getting Started with Qiskit Terra\n",
+    "\n",
+    "This tutorial provides an overview working with Qiskit Terra. Qiskit Terra provides the basic building blocks necessary to program quantum computers. The basic concept of Qiskit Terra is an array of quantum circuits. A workflow using Terra consists of two stages: Build and Execute. Build allows you to make different quantum circuits that represent the problem you are solving, and Execute allows you to run them on different backends.  After the jobs have been run, the data is collected. There are methods for putting this data together, depending on the program. This either gives you the answer you wanted, or allows you to make a better program for the next instance (e.g., when running the variational quantum eigensolver (VQE) algorithm).\n",
```

Comment:
Can we not use tutorial as these are imported into the html. This section or something.

Reply:
I'll make the changes.

At some points in this notebook I refer the reader to other notebooks. Do you know of some way we could do cross-referencing between the notebooks such that we'd be able to do hyperlinking between HTML pages?